sdcm/utils.py

```diff
@@ -104,3 +104,47 @@ def inner(*args, **kwargs):
                         logger.error("Number of retries exceeded!")
                         raise
         return inner
+
+
+def log_run_info(arg):
+    """
+        Decorator that prints BEGIN before the function runs and END when function finished running.
+        Uses function name as a name of action or string that can be given to the decorator.
+        If the function is a method of a class object, the class name will be printed out.
+
+        Usage examples:
+            @log_run_info
+            def foo(x, y=1):
+                pass
+            In: foo(1)
+            Out:
+                BEGIN: foo
+                END: foo (ran 0.000164)s
+
+            @log_run_info("Execute nemesis")
+            def disrupt():
+                pass
+            In: disrupt()
+            Out:
+                BEGIN: Execute nemesis
+                END: Execute nemesis (ran 0.000271)s
+    """
+    def _inner(func, msg=None):
+        @wraps(func)
+        def inner(*args, **kwargs):
+            class_name = ""
+            if len(args) > 0 and func.__name__ in dir(args[0]):
+                class_name = " <%s>" % args[0].__class__.__name__
+            action = "%s%s" % (msg, class_name)
+            start_time = datetime.datetime.now()
+            logger.info("BEGIN: %s", action)
```

Comment:
original msgs are sent by self.log.debug(), here you switched it to INFO level. Is it expected?